app-config.yaml

```diff
@@ -150,8 +150,15 @@ catalog:
   rules:
     - allow: [Component, System, Group, Resource, Location, Template, API]
   locations:
-    - type: url
-      target: https://github.com/redhat-developer/rhdh/blob/main/catalog-entities/all.yaml
+    - type: file
+      target: ../../catalog-info.yaml
+      rules:
+        - allow: [Component, System]
+
+    - type: file
+      target: ../../catalog-entities/all.yaml
```

Comment:
When testing this change, I get 
```
{"entity":"location:default/generated-28cca00eef20648da516bda20fe0e71b7b7364ba","level":"\u001b[33mwarn\u001b[39m","location":"file:/opt/catalog-entities/all.yaml","message":"file /opt/catalog-entities/all.yaml does not exist","plugin":"catalog","service":"backstage","timestamp":"2025-02-26 11:24:42"}
```

I think that it should not go two-levels up (`../../`).

When I tried to search for `all.yaml` or `catalog-info.yaml` in the container image I haven't found anything. This might require updating Dockerfile to add the files into the  container iamge